addons/mail/static/src/messaging/component/chatter/chatter.js

```diff
@@ -77,11 +77,25 @@ class Chatter extends Component {
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onAttachmentsChanged() {
+        if (this.chatter.hasRecordReloadOnAttachmentCreated) {
+            this.trigger('o-reload');
+        }
```

Comment:
I would just always trigger the event and let the parent handle it if he wants to. Then the prop can be removed?

Reply:
That would mean we depends on `this.chatterFields` in `form_renderer`, I don't know if this is what we want. This would change if we change public API.

Reply:
And that means it cannot be changed dynamically within owl (just an observation I don't know if it's wanted or not)

Reply:
(I'm ok with both solution, btw)